apps/web/next.config.js

```diff
@@ -77,6 +77,7 @@ module.exports = withTreat(
         graphqlUrl: '',
         graphqlEndpoint: graphqlPath,
         SENTRY_DSN,
+        disableApiCatalog: true,
```

Comment:
Should this maybe be an environment variable so we can have these pages accessible on dev and staging for testing?

Reply:
Ah yeah, that's a good point. I'll fix that.